### PR TITLE
Utilize the path instead of subdomain to manage multi-tenancy

### PR DESCRIPTION
**Pull request by edwinthinks**
### Describe your change in plain English.

Uses the `/path` instead of the subdomain to manage the multi-tenancy look up. This should be more sustainable in the long run using this approach.

https://jam.dev/c/492ce5db-b7f6-475e-a31e-b11ba92ea979

Notably a number of tests break using this. Maybe we can revisit which tests make sense as we are rebuilding a bunch of things.

### Link to the issue
https://github.com/rubyforgood/pet-rescue/issues/177

### CODE REVIEW

**Comment by kasugaijin** [comment]
I will leave this to someone with more experience in tenant architecture than me to approve, but this seems to be a reasonable and slightly simpler approach.

**Comment by edwinthinks** [comment]
> I will leave this to someone with more experience in tenant architecture than me to approve, but this seems to be a reasonable and slightly simpler approach.

Sure. In addition, I can work through and talk through this together in our Tuesday meeting. Most of the multi-tenancy comes from the acts_as_tenant gem, this really handles how we define it. Happy to talk through it.
